etl/data_enhancement/enhance_data.py

```diff
@@ -37,29 +41,41 @@ def run(self):
 
     def __run_for_domain(self, domain):
         """ run domainspecific enhancement. """
-        print(f"Run domain specific enhancement enhancement for {domain}")
+        self.logger.debug("__run_for_domain()")
+        self.logger.info(f"Run domain specific enhancement enhancement for {domain}")
+
         if domain in self.__function_map:
             self.__function_map[domain]()
         else:
-            print(f"Error [enhance_data.py]: No function set for '{domain}'")
+            self.logger.error(f"No function set for '{domain}'")
 
     def __enhance_ehrenamt_hessen(self):
         """ domain specific enhancement for ehrenamtsuche hessen """
+        self.logger.debug("__enhance_ehrenamt_hessen()")
+
         e_tags.run(self.__data, self.__domain_name)
 
     def __enhance_weltwaerts(self):
         """ domain specific enhancement for weltwaerst """
+        self.logger.debug("__enhance_weltwaerts()")
+
         pass
```

Comment:
The pass can be removed. It is only needed if a function would be empty without it. Maybe instead you can dabug a message that no enhancements are needed.